src/dosbox.cpp

```diff
@@ -773,11 +773,14 @@ static void add_dosbox_config_section(const ConfigPtr& conf)
 
 	pstring->SetHelp(
 	        "Select the DOS messages language:\n"
+	        "\n"
```

Comment:
1. If you really need such formatting change, I suggest to do it in a separate PR, for all the sections - for consistency.
2. It would be nice to trim the comments before writing the config file, currently such formatting causes spaces at the end of the line - here is how my editor displays the configuration now:

<img width="1718" height="960" alt="Screenshot" src="https://github.com/user-attachments/assets/0fadabff-b463-4578-a77a-21ee5cbe78c8" />

Reply:
Removed the commit.